.github/CONTRIBUTING.md

```diff
@@ -5,189 +5,297 @@ If you at any point have any questions please read: [I have a question related t
+
+1. Clone the repository: 
+
+``` 
+
+git clone https://github.com/kirbydesign/designsystem
+```
+
+2. Install dependencies: 
 
-### Clone and install repository 
-- how to clone and get going with the kirby repo
+``` 
+
+npm install
+```
+
+3. Start the cookbook server:
+
+``` 
+
+npm start
+```
+
+You should now be able to open `http://localhost:4200` in a browser to access the cookbook.
 
 ### Node 
-- Which version should people use, and perhaps a recommendation on which tool people can use to manage versions. 
+
+<!-- Which version should people use, and perhaps a recommendation on which tool people can use to manage versions. -->
+[Node](https://nodejs.org/en/) is the runtime used for development. 
+
+It is recommended to use `v12` for development as we currently do not support node LTS (see issue: [#1443](https://github.com/kirbydesign/designsystem/issues/1443)). 
+
+If you need to have multiple versions of node installed then a version manager such as [NVM](https://github.com/nvm-sh/nvm) can be used. 
 
 ### Tslint 
-- People should make sure to have it enabled to avoid unnecessary comments during code review
+
+<!-- People should make sure to have it enabled to avoid unnecessary comments during code review-->
+**Note:** Work is being done to migrate from TSLint to ESLint (see issue: [#1317](https://github.com/kirbydesign/designsystem/issues/1317)). 
+
+TSLint is used for linting which helps keep up the quality of the submitted code for Kirby. 
+
+Scripts for running TSLint can be found in `package.json` . We however recommend that you install a TSLint plugin for your editor. It can be useful to enable "fix on save"-functionality such that simple errors are fixed automatically. 
+
+**Useful resources:**
+
+* [TSLint extension for VSCode](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin).
+* [Enabling "fix on save" in VSCode](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin#auto-fix-and-auto-fix-on-save).
+
+#### VSCode "workspace library execution"-issue
+
+An issue might occur where TSLint will randomly disable for the current workspace and has to be enabled again using the command `TSLint: Manage workspace library execution` . 
+
+To solve this: 
+
+1. open a typescript file in VScode. 
+2. open the command pallette (<kbd>ctrl/cmd</kbd> + <kbd>shift</kbd> + <kbd>p</kbd>) and enter: <br />
+
+`typescript: select typescript version` .
+
+4. select: `use workspace version 3.6.5`.
+5. open the command pallette and enter: <br />
+
+`TSLint: Manage workspace library execution` .
+
+6. select: `always enable workspace library execution`.
+
+This will cause changes to be made in the file `.vscode/settings.json` ; do not commit these.
 
 ### Prettier
-- It is a good idea to use format-on-save
+
+Prettier is used for consistent code formatting and avoiding unnecessary discussions about this during reviews.
```

Comment:
Consider adding a sentence about reducing "diff clutter".

When everything is formatted the same way using Prettier there should be no changes that are "formatting only".

Reply:
How's this? (something 

<hr />

As with TSLint, we  strongly recommend installing Prettier as a plugin for your editor. You should also enable "format on save".

It can however be used without an editor plugin; run `npm run prettier` to check for formatting errors, fix them by running `npm run prettier:fix`. 

If there are changes only concerning formatting, in files you did not create - do not commit these. There is most likely something wrong with your Prettier settings. 

Make sure that Prettier uses the config file that is shipped with the repository ([`.prettierrc.json`](https://github.com/kirbydesign/designsystem/blob/master/.prettierrc.json)) and not a global configuration located on your machine.

Reply:
👍